workspaces/frontend/src/app/pages/Workspaces/Details/WorkspaceDetails.tsx

```diff
@@ -72,7 +73,9 @@ export const WorkspaceDetails: React.FunctionComponent<WorkspaceDetailsProps> =
             aria-label="Pod template"
           >
             <TabContent id="podTemplateBodyPadding">
-              <TabContentBody hasPadding>Pod template</TabContentBody>
+              <TabContentBody hasPadding style={{ height: '500px' }}>
```

Comment:
```suggestion
              <TabContentBody hasPadding>
```

Inline style can be removed once the suggested fix is applied.

Reply:
I removed it, thanks!